pvanalytics/metrics.py

```diff
@@ -36,11 +36,12 @@ def performance_ratio_nrel(poa_global, temp_air, wind_speed, pac, pdc0,
     wind_speed : numeric
         Wind speed at a height of 10 meters [m/s].
 
-    pac : float
+    pac : numeric
         AC power [kW].
 
     pdc0 : float
         Power of the modules at 1000 W/m2 and cell reference temperature [kW].
+        Otherwise referred to as the PV array STC nameplate rating [kW].
```

Comment:
```suggestion
        Power of the modules at 1000 W/m2 and cell reference temperature,
        otherwise referred to as the PV array STC nameplate rating [kW].
```